docs/plugins/parentwork.rst

```diff
@@ -0,0 +1,46 @@
+Parentwork Plugin
+=================
+
+The ``parentwork`` plugin fetches the work title, parentwork title and 
+parentwork composer. 
```

Comment:
I know the plugin is spelled `parentwork`, but maybe we can use "parent work" when discussing the concept?